install-proto-osx.sh

```diff
@@ -9,11 +9,15 @@ wget https://github.com/grpc-ecosystem/grpc-gateway/releases/download/v1.9.6/pro
     && chmod +x protoc-gen-swagger-v1.9.6-darwin-x86_64 \
     && sudo ln -s -f ${DIR}/protoc-gen-swagger-v1.9.6-darwin-x86_64 /usr/local/bin/protoc-gen-swagger
 
+wget https://github.com/grpc-ecosystem/grpc-gateway/releases/download/v1.9.6/protoc-gen-grpc-gateway-v1.9.6-darwin-x86_64 \
```

Comment:
The latest release is `v1.13.0`, but I wanted to keep it consistent with `protoc-gen-swagger` above.
Could bump both if there is no objection.

Reply:
I think it is fine to bump both.

Reply:
Will do separately to see if there any side effects of that.

Reply:
Please consider to move this changes to bump protoc version in a separate PR.

Reply:
I didn't change version, I added installation of the new `protoc-gen-grpc-gateway` binary that is required for generating gRPC gateway boilerplate, matching the version of existing `protoc-gen-swagger`.